modules/nvidia-gpu-features.adoc

```diff
@@ -54,5 +54,5 @@ Up until this point, the GPU Operator only provisioned worker nodes to run GPU-a
 +
 You can configure the GPU Operator to deploy different software components to worker nodes depending on which GPU workload is configured to run on those nodes.
 
-GPU Operator dashboard::
-You can install a console plugin to display GPU usage information on the cluster utilization screen in the {product title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts) for each GPU and the percentage of GPU workload used for video encoding and decoding.
+GPU Monitoring dashboard::
+You can install a monitoring dashboard to display GPU usage information on the cluster Observe screen in the {product-title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts), temperature (in degrees Celsius), utilization (in percent), and other metrics for each GPU.
```

Comment:
```suggestion
You can install a monitoring dashboard to display GPU usage information on the cluster *Observe* page in the {product-title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts), temperature (in degrees Celsius), utilization (in percent), and other metrics for each GPU.
```
Literal web console elements like page names should be in bold. Also, per the IBMSG, please use page instead of screen.
We use "page" in other places in the docs that refer to the web console. For example: https://docs.openshift.com/container-platform/4.13/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-creating-sample-applications_odc-creating-applications-using-developer-perspective

![image](https://github.com/openshift/openshift-docs/assets/40172997/bff1796e-3dec-4ddb-9fe7-1cd1f3eab62b)

Reply:
The fix is applicable to 4.13 and on. What do you mean 4.13+? There's no documentation for 4.14 yet.

Reply:
Also, would it be OK if I submit a fix for your comment in a separate PR? This one is a month old, and I don't think we have time for another cycle of reviews, notifications, approvals etc., as the customers keep landing on the wrong page and trying to use the deprecated software.

Reply:
@abrennan89 This change is for 4.13 and all future versions.

Reply:
Thanks folks!